include/LVSS.hpp

```diff
@@ -1,13 +1,58 @@
-#pragma once
+#ifndef _LVSS_
```

Comment:
I think the standard is to use the define flag LVSS_HPP, but I guess it doesn't really matter.

Reply:
im just gonna leave it because i dont feel like changing it if it doesn't matter